components/CtaImageContent/src/index.scss

```diff
@@ -63,9 +71,17 @@
   margin-block-end: var(--denhaag-cta-image-content-text-margin-block);
   color: var(--denhaag-cta-image-content-text-color);
   font-weight: var(--denhaag-cta-image-content-text-font-weight);
-  font-size: var(--denhaag-cta-image-content-text-font-size);
-  line-height: var(--denhaag-cta-image-content-text-line-height);
   font-family: var(--denhaag-typography-sans-serif-font-family);
+
+  &:not(&.denhaag-cta-image-content__text--small) {
```

Comment:
The second `&` (in the `:not()`) is not necessary here, right?

`.denhaag-cta-image-content__text:not(.denhaag-cta-image-content__text--small)` selects exactly the same as the current `.denhaag-cta-image-content__text:not(.denhaag-cta-image-content__text.denhaag-cta-image-content__text--small)`.